uiowa/uiowa
# Add bigmenu module

relates to #1996

## To Test

- pull down a site with more than one-level deep main menu. cnm.uiowa.edu currently has a large menu.
- ensure the module is enabled and that you see the edit children column and links within the menu management interface
- notice how the site doesn't error if you try to modify a menu link via this interface.
- test to see if the usability is still adequate while providing error protection
